scalameter-core/src/test/scala/org/scalameter/picklers/PicklersSpecification.scala

```diff
@@ -0,0 +1,89 @@
+package org.scalameter.picklers
+
+import java.util.Date
+import org.scalameter.picklers.Implicits._
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{FunSuite, Matchers}
+
+
+class PicklerSpecification extends FunSuite with PropertyChecks with Matchers {
+  def validatePickler[T: Pickler](o: T) = implicitly[Pickler[T]].unpickle(implicitly[Pickler[T]].pickle(o)) should === (o)
+
+  test("Byte pickling") {
+    forAll { o: Byte =>
```

Comment:
It's definitely possible to change some of this parameters - see "Property change configuration" on [scalatest user guide](http://www.scalatest.org/user_guide/generator_driven_property_checks).
I'm not sure what with e.g. random value generator seed - I'll check that.